hive-plugin/src/main/java/com/wipro/ats/bdre/hiveplugin/hook/LineageHook.java

```diff
@@ -53,34 +55,26 @@ public void run(HookContext hookContext) {
 				    String key = (String)(entry.getKey());
 				    if (LineageConstants.processIdString.equalsIgnoreCase(key)) {
 					    processId = (String) (entry.getValue());
-					    System.out.println("ProcessId found in Hive = " + processId);
+					    LOGGER.info("ProcessId found in Hive = " + processId);
 				    } else if (LineageConstants.instanceExecIdString.equalsIgnoreCase(key)) {
 					    instanceExecId = (String) (entry.getValue());
-					    System.out.println("InstanceExecId found in Hive = " + instanceExecId);
+					    LOGGER.info("InstanceExecId found in Hive = " + instanceExecId);
 				    }
 			    }
 
-//			    if (processId == null) {
-//				    System.out.println("Error: bdre.lineage.processId is not supplied while calling Hive. Assuming default value");
-//				    processId = "" + LineageConstants.processId;
-//			    }
-//			    if (instanceExecId == null) {
-//				    System.out.println("Error: bdre.lineage.instanceExecId is not supplied while calling Hive. Assuming default value");
-//				    instanceExecId = "" + LineageConstants.instanceId;
-//			    }
 			    if (processId == null || instanceExecId == null) {
-				    System.out.println("Warning: ProcessId or InstanceExecId is null. Lineage not done. End of Lineage Hook.");
+				    LOGGER.info("Warning: ProcessId or InstanceExecId is null. Lineage not done. End of Lineage Hook.");
 				    return;
 			    }
 
 		    }
-		    System.out.println("BDRE Printed query = " + wholeQuery);
+		    LOGGER.info("BDRE Printed query = " + wholeQuery);
 
 		    LineageProcessor lineageProcessor = new LineageProcessor();
 		    lineageProcessor.execute(wholeQuery, processId, instanceExecId);
 			successFlag = true;
 
-	    } catch (Throwable e) {
+	    } catch (Exception e) {
 		    LOGGER.error("Error in executing LineageHook", e);
 			e.printStackTrace(System.out);
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Use a logger to log this exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1148)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Replace this usage of System.out or System.err by a logger. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS106)